lib/tasks/complexity.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var complexity = require('../pipelines/complexity');
+var normalizeOptions = require('../options');
+var path = require('path');
+
+module.exports = function(options) {
+	options = normalizeOptions(options);
+	var gulp = options.gulp;
+	var taskPrefix = options.taskPrefix;
+
+	gulp.task(taskPrefix + 'complexity:report', function() {
+		return gulp.src(options.complexityGlobs)
```

Comment:
We always use the name `Src` for this kind of option, so to keep it consistent this should be `complexitySrc` instead. Can you rename it?